UI/DragNDrop.cpp

```diff
@@ -3,27 +3,28 @@
 
 #define TIME_TO_GO_BACK_TO_ORIGIN 0.025
```

Comment:
did you forget to define the static bool in this file? I do not see it

Reply:
added to constructor - though it worked fine without this

Reply:
I meant putting `bool DragNDrop::m_focused = false;` at the top of the file, not in the constructor -- does the game build ?

Reply:
Also don't want this in the constructor as a new card could be created while one is being dragged and screw things up